packages/itwin/reports-config-widget/src/widget/components/Reports.tsx

```diff
@@ -107,6 +114,35 @@ export const Reports = () => {
     [reports, searchValue]
   );
 
+  useEffect(() => {
+    if (jobStarted) {
+      const interval = window.setInterval(async () => {
+        const states = await bulkExtractor.getStates(reports.map((r) => r.id));
+        if (Array.from(states.values())
+          .filter((x) => x === ExtractionStates.Succeeded ||
+            x === ExtractionStates.Failed ||
+            x === ExtractionStates.None)
+          .length === states.size) {
+          setJobStarted(false);
+          bulkExtractor.clearJobs();
+        }
+        setExtractionStates(states);
+      }, 5000);
```

Comment:
Try to use less magic numbers. Move it to some constants file.

Reply:
`Constants.ts` file has been added with constant variable `STATUS_CHECK_INTERVAL`